### ISSUE TITLE
AdGuard in split screen is clipping thru the buttons

### ISSUE THREAD

**Issue by JustAAdGuardFan** [AdguardTeam/AdguardForAndroid#5325]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard version

4.4 Nightly 47

### Environment

- OS version: Android 13
- Device: Galaxy A71

### HTTPS filtering

- [X] yes, I do

### Root access

- [ ] yes, I have it

### Integration with AdGuard VPN

- [ ] yes, I do

### Routing mode

Local VPN

### Ad Blocking

AdGuard Base filter, AdGuard Mobile Ads filter, EasyList

### Social

AdGuard Social Media filter, Fanboy's Social Blocking List

### Annoyances

AdGuard Annoyances filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Mobile App Banners filter, AdGuard Other Annoyances filter, AdGuard Widgets filter, Adblock Warning Removal List, Fanboy's Annoyances, EasyList Cookie List, Dandelion Sprout's Annoyances List

### Security

Online Malicious URL Blocklist, NoCoin Filter List

### Other

Filter unblocking search ads and self-promotion, AdGuard DNS filter, AdGuard Experimental filter

### Which DNS server do you use?

AdGuard DNS

### DNS protocol

DNS-over-HTTPS

### Issue Details

Steps to reproduce:
1. A app + AdGuard in split screen
2. Done

### Expected Behavior

Adguard in split screen is not clipping thru the buttons

### Actual Behavior

Adguard in split screen is clipping thru the buttons

### Screenshots

<details><summary>Screenshot 1</summary>

![Screenshot_20240518_002010_AdGuard](https://github.com/AdguardTeam/AdguardForAndroid/assets/162761198/6c5ed78c-e7d6-4fc1-b4a9-08a6305e6a61)

</details>

### Additional Information

 *

**Comment by Versty** [comment]
@JustAAdGuardFan We are aware of this issue, will be resolved in future versions. 

Duplicate of https://github.com/AdguardTeam/AdguardForAndroid/issues/4686
